test/server.js

```diff
@@ -79,18 +79,21 @@ t.test('redactThrow', async t => {
   await t.test('successfully throws error', async t => {
     const badError = new CustomError('hello world', 'sensitive data')
     t.same(badError.sensitive, 'sensitive data', 'should have sensitive field')
+
+    const handler = async () => {
+      throw badError
+    }
```

Comment:
Optional: Test arguments are passed through by using an argument to construct badError's message maybe?

Reply:
love it, added a test